addons/calendar/models/calendar_event.py

```diff
@@ -688,11 +688,15 @@ def write(self, values):
 
         return True
 
+    def _get_trigger_alarm_types(self):
+        return ['email']
+
     def _setup_alarms(self):
         """ Trigger the cron in the future for every email reminder """
         cron = self.env.ref('calendar.ir_cron_scheduler_alarm')
+        tat = self._get_trigger_alarm_types()
```

Comment:
"alarm_triggers" ? tat seems a bit too much like the gloomy 'ptav' and 'ptal' to me :D 
I would prefer explicit naming